.gitlab-ci.yml

```diff
@@ -0,0 +1,65 @@
+variables:
+  ALPAKA_CI_OS_NAME: "Linux"
+  ALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLE: "ON"
+  ALPAKA_ACC_CPU_B_SEQ_T_THREADS_ENABLE: "ON"
+  ALPAKA_ACC_CPU_B_SEQ_T_FIBERS_ENABLE: "OFF"
+  ALPAKA_ACC_CPU_B_TBB_T_SEQ_ENABLE: "OFF"
+  ALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLE: "ON"
+  ALPAKA_ACC_CPU_B_SEQ_T_OMP2_ENABLE: "ON"
+  ALPAKA_ACC_ANY_BT_OMP5_ENABLE: "OFF"
+  ALPAKA_ACC_ANY_BT_OACC_ENABLE: "OFF"
+  ALPAKA_ACC_GPU_CUDA_ENABLE: "OFF"
+  ALPAKA_ACC_GPU_CUDA_ONLY_MODE: "OFF"
+  ALPAKA_ACC_GPU_HIP_ENABLE: "OFF"
+  ALPAKA_ACC_GPU_HIP_ONLY_MODE: "OFF"
+  # if OFF, run tests
```

Comment:
please explain it more e.g. `If variable is OFF compile and execute runtime tests else compile only.`